docs/source/apriltag/vision_portal/apriltag_advanced_use/apriltag-advanced-use.rst

```diff
@@ -90,6 +91,67 @@ Further discussion is provided here:
 
 - https://ftc-docs.firstinspires.org/apriltag-detection-values
 
+Advanced CPU Management
+-----------------------
+
+This section continues from the **VisionPortal CPU and Bandwidth** page, which
+covered many basic tools for avoiding limits of CPU usage and USB bandwidth.
+
+To evaluate multiple factors, changing at the same time, a customized Test
+OpMode can be very useful.  This section provides an example that allows **live
+gamepad control** to:
+
+- toggle AprilTag Processor on and off
+- toggle TFOD Processor on and off
+- toggle LiveView on and off
+- toggle Streaming on and off
+
+Other features of this Test OpMode include:
+
+- All controls are independent, to explore the combinations and their effect on
+  frame rate (FPS).
+- The previews can be observed, and detections/recognitions can be monitored
+  via annotations and Telemetry.
+- Frame rate is provided in LiveView and DS Telemetry.
+- The Telemetry functions include an alternate for getting **all** or **only
+  fresh** detections/recognitions.
+
+This Test OpMode can be downloaded for `FTC
+Blocks <https://gist.github.com/WestsideRobotics/ea7540175b11d07bf362cb9625a54042>`__
+or
+`Java <https://gist.github.com/WestsideRobotics/8566a1148dee6572084a6dea2b0d5627>`__.
+The Blocks version is shown below; right-click to open in a new browser tab and
+zoom in.
+
+.. figure:: images/100-VisionPortal-Test-v01.png
+   :width: 75%
+   :align: center
+   :alt: VisionPortal Example Code
+
+   VisionPortal Example OpMode
+
+The OpMode uses "Webcam 1", or change ``USE_WEBCAM`` for a built-in RC phone
+camera.  For Control Hub, set up an HDMI monitor or `scrcpy
+<https://github.com/Genymobile/scrcpy>`__.  Follow the DS gamepad button guide.
+
+At that **VisionPortal CPU and Bandwidth** page, four tools mentioned were not
+discussed:
+
+- set decimation (down-sampling)
+- select a pose solver algorithm
+- get all or only fresh detections from the AprilTag Processor
+- get all or only fresh recognitions from the TFOD Processor
+
+For now, these are left for interested Blocks and Java users to research and
+investigate.  In time, more information may be posted at this page.
+
+All of the above features are easily found in the relevant **FTC Blocks**
+toolbox, or palette, under the Vision category.
+
+**Java** users should review the VisionPortal interface at the `SDK Javadocs
+<https://javadoc.io/doc/org.firstinspires.ftc/RobotCore/latest/overview-summary.html>`__
```

Comment:
Shouldn't this point to the Vision library's javadoc instead of RobotCore?

Note that it can take a while for javadoc.io to scrape new maven versions.

Reply:
I guess? I don't know where that is. I'll make an issue to fix the link.